### ISSUE TITLE
Training accuracy converges very fast

### ISSUE THREAD

**Issue by shuzhiyu** [raghakot/keras-resnet#29]
Hello everyone, I have run the cifar10.py (set batch_size=128 as in the paper, and set data_augmentation=false, using sgd). My result shows that the training accuracy converges to 1 after 30 epochs, but the testing accuracy converges to 0.6. I do not understand why training accuracy converges so fast (much faster than the results in paper). Does anyone has an idea of the reason?

Thank you!

**Comment by raghakot** [comment]
You are overfitting. 
- Try a lower learning rate.
- Use l2 regularization.

**Comment by raghakot** [comment]
We made some changes to cifar10 script. Should work out of the box. Reopen if you are still seeing the issue.
